kernel/kselftest.py.data/futex.yaml

```diff
@@ -0,0 +1,9 @@
+component: !mux
+    futex:
+        comp: 'futex'
+
+run_type: !mux
+    upstream:
+        type: 'upstream'
+        location: 'git://git.kernel.org/pub/scm/linux/kernel/git/powerpc/linux.git'
+        branch: 'merge'
```

Comment:
are you interested on merge branch or master branch and check for URL

Reply:
Oh, I need the master branch i will update the changes